.travis.yml

```diff
@@ -5,22 +5,30 @@ cache:
     - node_modules
 notifications:
   email: false
-script:
-  - commitlint-travis
-  - yarn lint
-  - yarn test
-  - if [ "$TRAVIS_BRANCH" = "master" ]; then yarn docs; fi
-deploy:
-  - provider: pages
-    skip-cleanup: true
-    keep-history: true
-    github-token: $GH_TOKEN
-    local-dir: docs
-    on:
-      branch: master
-  - provider: script
-    skip-cleanup: true
-    script:
-      - .travis/release.sh
-    on:
-      branch: master
\ No newline at end of file
+jobs:
+  include:
+    - stage: "lint"
+      name: "Lint commits"
+      script: commitlint-travis
+    - script: yarn lint
+      name: "Lint code"
+    - stage: "tests"
+      name: "Unit tests"
+      script: "yarn test"
+    - stage: "docs"
+      name: "Generate documentation"
+      if: branch = master
+      deploy:
+        - provider: pages
+          skip-cleanup: true
+          keep-history: true
+          github-token: $GH_TOKEN
+          local-dir: docs
+    - stage: "deploy"
```

Comment:
looking at the docs if you name deploy and test you don't need quotes https://docs.travis-ci.com/user/build-stages

Reply:
on the documentations some times they use quotes some times they don't. I don't think it matters if we are consistent.